Dependency conflict with com.intellij:annotations:12.0

Hello! I am trying to implement Kaspresso in my project, but i have conflicts with core annotations library:

```FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':core:checkDebugAndroidTestDuplicateClasses'.
> 1 exception was raised by workers:
  java.lang.RuntimeException: Duplicate class org.intellij.lang.annotations.Identifier found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.JdkConstants found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.JdkConstants$AdjustableOrientation found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.JdkConstants$BoxLayoutAxis found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.JdkConstants$CalendarMonth found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.JdkConstants$CursorType found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.JdkConstants$FlowLayoutAlignment found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.JdkConstants$FontStyle found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.JdkConstants$HorizontalAlignment found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.JdkConstants$InputEventMask found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.JdkConstants$ListSelectionMode found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.JdkConstants$PatternFlags found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.JdkConstants$TabLayoutPolicy found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.JdkConstants$TabPlacement found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.JdkConstants$TitledBorderJustification found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.JdkConstants$TitledBorderTitlePosition found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.JdkConstants$TreeSelectionMode found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.Language found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.MagicConstant found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.Pattern found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.PrintFormat found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.PrintFormatPattern found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.RegExp found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.intellij.lang.annotations.Subst found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.jetbrains.annotations.Nls found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.jetbrains.annotations.NonNls found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.jetbrains.annotations.NotNull found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.jetbrains.annotations.Nullable found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.jetbrains.annotations.PropertyKey found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
  Duplicate class org.jetbrains.annotations.TestOnly found in modules adbserver-device-1.0.0.jar (com.kaspersky.android-components:adbserver-device:1.0.0) and annotations-12.0.jar (com.intellij:annotations:12.0)
```

I have tried to exclude org.jetbrains.annotations module from kaspresso dependency in implementing dependency process, but it couldn't help. Please, give me the way, how i can resolve that:)

My gradle code:
```
dependencies {
    androidTestImplementation libraries.junitAndroid
    androidTestImplementation( libraries.kaspresso ){
        exclude(group: 'com.intellij', module: 'annotations')
    }
    androidTestImplementation libraries.espressoCore
    androidTestImplementation libraries.uiAutomator
    androidTestImplementation libraries.kakao
    androidTestImplementation libraries.espressoWeb
    androidTestImplementation libraries.espressoContrib
    androidTestImplementation libraries.espressoIntents 
}
 ```

dependencies.gradle:
```
        junitAndroid            : "androidx.test.ext:junit:1.1.1",
        kaspresso               : "com.kaspersky.android-components:kaspresso:1.0.1",
        espressoCore            : "androidx.test.espresso:espresso-core:3.2.0",
        uiAutomator             : "androidx.test.uiautomator:uiautomator:2.2.0",
        kakao                   : "com.agoda.kakao:kakao:2.2.0,
        espressoWeb             : "androidx.test.espresso:espresso-web:3.2.0",
        espressoContrib         : "androidx.test.espresso:espresso-contrib:3.2.0",
        espressoIntents         : "androidx.test.espresso:espresso-intents:3.2.0",
```

--- a/dependencies.gradle
+++ b/dependencies.gradle
@@ -20,7 +20,7 @@ ext.versions = [
         dokka          : "0.9.18",
         kakao          : "2.2.0",
         gson           : "2.8.5",
-        adbserver      : "1.0.0",
+        adbserver      : "1.0.1",
         truth          : "1.0",
         material       : "1.0.0",
 ]

--- a/kaspresso/src/main/kotlin/com/kaspersky/kaspresso/device/files/Files.kt
+++ b/kaspresso/src/main/kotlin/com/kaspersky/kaspresso/device/files/Files.kt
@@ -42,6 +42,6 @@ interface Files {
      * @param devicePath a file path relative to the device directory.
      * @param serverPath a path to copy. (If empty - pulls in adbServer directory (folder with file "desktop.jar"))
      */
-    @AdbServerMust
+    @RequiresAdbServer
     fun pull(devicePath: String, serverPath: String = "")
 }

